src/qutip_jax/reshape.py

```diff
@@ -64,14 +63,14 @@ def split_columns_jaxarray(matrix, copy=None):
 
 
 def _parse_ptrace_inputs(dims, sel, shape):
-    dims = np.atleast_1d(dims).ravel()
-    sel = np.atleast_1d(sel)
+    dims = jnp.atleast_1d(dims).ravel()
```

Comment:
I think since jnp is not defined in the header nothing will happen, maybe you should change jnp to JaxArray or import jax.numpy as jnp